wireapp/wire-android
# feat: Deleted users adjustments [WPB-1655]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1655" title="WPB-1655" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-1655</a>  Issues with deleted users
  </td></table>
  <br />
 


https://wearezeta.atlassian.net/browse/WPB-1655

# What's new in this PR?

### Issues

We were able to perform actions on deleted users, block them, change notifications, send invitation.

### Solutions

Use `blocked` flag to hide those actions

#### How to Test

- Delete user
- Check if you can open his profile - you shouldnt
- Check if longtapping on conversation gives the option. to change notifications and blocking - you shouldnt 
- Search for non deleted user, open its profile, then delete the user
- Options to invite, blockm and change notifications should also not be present

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
